components/Layout.js

```diff
@@ -17,8 +17,9 @@ export default ({ children }) => {
 
         <meta
           property="og:image"
-          content="http://copenhagenjs.dk/static/images/logo.png"
+          content="http://copenhagenjs.dk/static/images/logo-small.png"
         />
+        {/* TODO: use FeedBurner */}
```

Comment:
So long since I used Feedburner, is it still relevant? :D

Reply:
I don't know, I just copied your comment from one of the files to be deleted
 @kevinsimper are you planning on keeping RSS Feeds?